global.json

```diff
@@ -5,10 +5,5 @@
   "msbuild-sdks": {
     "MSBuild.Sdk.Extras": "3.0.44",
     "Microsoft.Build.NoTargets": "3.3.0"
-  },
-  "sdk": {
-    "version": "7.0.200",
-    "allowPrerelease": true,
-    "rollForward": "major"
```

Comment:
Removing these lines is very similar behavior to it being here. `rollForward=major` allows it to use .NET 8 SDKs.

Either way, seems fine to just delete these as even the .NET team doesn't really suggest using `global.json` by default -- it's more of something for working around a problem.

Reply:
Yea, I was thinking with your edits it should be using .NET8.

But mine was still using .NET 7 when I had these there. I had to delete them in order for VS or the CLI to use .NET8.

![image](https://github.com/dotnet/maui/assets/5375137/7ad0b175-5edd-4f4c-ae10-50b7f0b3fa7d)

![image](https://github.com/dotnet/maui/assets/5375137/4047fc80-647f-4bc8-a101-5da6172b00f6)